setup.py

```diff
@@ -38,7 +38,7 @@
       keywords='mozilla services',
       author='Mozilla Cloud Services QA Team',
       author_email='cloud-services-qa@mozilla.com',
-      url='https://github.com/mozilla-services/services-marionette',
+      url='https://github.com/mozilla-services/marionette-wrapper',  # noqa
```

Comment:
Why does this need the `#noqa` bit? (not sure what flake warning would be solving since the line doesnt look too long)